auth/email_action_links.go

```diff
@@ -32,6 +32,7 @@ type ActionCodeSettings struct {
 	AndroidMinimumVersion string `json:"androidMinimumVersion,omitempty"`
 	AndroidInstallApp     bool   `json:"androidInstallApp,omitempty"`
 	DynamicLinkDomain     string `json:"dynamicLinkDomain,omitempty"`
+	ReturnOobLink         bool   `json:"returnOobLink"`
```

Comment:
The naming for the field is a bit weird. Since we want to keep the default value of the boolean as false, it would be better to rename this variable to `SendEmailLink` instead.